GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -1024,6 +1026,11 @@ public static ArgumentString MergeBranchCmd(string branch, bool allowFastForward
             };
         }
 
+        private static string EscapeDoubleQuotes(string message)
```

Comment:
Should rather be named `EscapeQuotes`.
Code block can be omitted, i.e. turn into `=> message?.Replace("\"", "\\\"");`